site2/website/scripts/replace.js

```diff
@@ -22,11 +22,11 @@ function downloadPageUrl() {
 }
 
 function binaryReleaseUrl(version) {
-  return `http://www.apache.org/dyn/closer.cgi/incubator/pulsar/pulsar-${version}/apache-pulsar-${version}-bin.tar.gz`
```

Comment:
why are you making this change? I suspect the asf ops folks would frown on this.

Reply:
archive is okay, it just can't be dist directly. all the links in our download pages for old releases are from archive.

closer.cgi is a redirection to a page showing what is the closer mirror. but it doesn't download directly. so when it is used in command line,  it doesn't really work. I noticed this when I am trying out all the quickstart in pulsar documentation. so this leads me to change it to archive. do you know any better links here?

Reply:
ah, that does suck.